Formula/clamav.rb

```diff
@@ -68,5 +69,6 @@ def caveats
 
   test do
     system "#{bin}/clamav-config", "--version"
+    system "#{bin}/freshclam", "-V"
```

Comment:
Is there a way to maybe get virusdefinitions and check if that succeeded?

Reply:
I'm sorry I don't know. Someone has to step in here, please.

Reply:
The following lines will update definitions and then scan them.
```suggestion
    system "#{bin}/freshclam", "--datadir=#{testpath}"
    system "#{bin}/clamscan", "--database=#{testpath}", testpath
```